src/main/kotlin/software/amazon/smithy/intellij/index/SmithyBuildConfigurationIndex.kt

```diff
@@ -47,9 +47,17 @@ class SmithyBuildConfigurationIndex : SingleEntryFileBasedIndexExtension<SmithyB
     override fun getValueExternalizer(): DataExternalizer<SmithyBuildConfiguration> =
         object : DataExternalizer<SmithyBuildConfiguration> {
             override fun save(out: DataOutput, config: SmithyBuildConfiguration) =
-                out.writeUTF(SmithyJson.writeValueAsString(config))
+                out.writeBytes(SmithyJson.writeValueAsString(config))
 
-            override fun read(`in`: DataInput) = SmithyJson.readValue<SmithyBuildConfiguration>(`in`.readUTF())
+            override fun read(`in`: DataInput): SmithyBuildConfiguration {
+                var line: String? = `in`.readLine()
+                var value = ""
+                while (line != null) {
+                    value += line
+                    line = `in`.readLine()
+                }
+                return SmithyJson.readValue<SmithyBuildConfiguration>(value)
+            }
         }
 
     override fun getVersion() = 2
```

Comment:
Index versions need to get bumped to force re-indexing. 

Otherwise, anyone with a project that had a build config large enough to cause exceptions wouldn't get this fix until they invalidated caches.

Reply:
I ended up changing the version - it did cause Intellij to immediately re-index the file, which was useful when I was testing. Previously I had to wait a few minutes for it to index it